packages/paste-core/components/corner-ornament/src/CornerOrnament.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, BoxStyleProps, safelySpreadBoxProps } from "@twilio-paste/box";
+import { IconSizeOptions } from "@twilio-paste/style-props";
+import * as React from "react";
+
+import { useCornerOrnamentContext } from "./CornerOrnamentContext";
+import { CornerOrnamentPosition, CornerOrnamentProps, CornerOrnamentType } from "./types";
+
+type OrnamentSpacingMapping = Record<
+  CornerOrnamentType,
+  Record<CornerOrnamentPosition, Partial<Record<IconSizeOptions, BoxStyleProps>>>
+>;
+
+export const CornerOrnament = React.forwardRef<HTMLDivElement, CornerOrnamentProps>(
+  ({ element = "CORNER_ORNAMENT", ...props }, ref) => {
```

Comment:
thinking of adding a prop to allow users to set the top and left properties if for whatever reason it doesn't look right with our mappings. Thoughts?

Reply:
hmm let's play around with as many different base/element combinations as possible but this isn't a bad idea. we could also wait until someone needs it

Reply:
resolved with no props exposed